noxfile.py

```diff
@@ -534,8 +526,12 @@ def load_pkg_versions():
 
 @nox.session(name="ffi-check")
 def ffi_check(session: nox.Session):
-    session.run("cargo", "doc", "-p", "pyo3-ffi", "--no-deps", external=True)
-    _run(session, "cargo", "run", "-p", "pyo3-ffi-check", external=True)
+    path = ["--manifest-path", "pyo3-ffi-check/Cargo.toml"]
+    session.run("cargo", "doc", *path, "-p", "pyo3-ffi", "--no-deps", external=True)
+    _run(
+        session, "cargo", "clippy", "--workspace", "--all-targets", *path, external=True
```

Comment:
This one I do not understand. This is to make sure the ffi-check binary is Clippy clean? But why the `--workspace` flag then?

Reply:
Yes, to check the ffi-check binary, and then `--workspace` is to also check `pyo3-ffi-check/macro`. I've force-pushed to add "macro" subdirectory to the workspace section in `pyo3-ffi-check/Cargo.toml`.

Also, good point about rustfmt - I've added that to the `fmt-rust` job.